fabric-common/lib/DiscoveryService.js

```diff
@@ -121,17 +125,21 @@ class DiscoveryService extends ServiceAction {
 	 *  ]
 	 * @example <caption>"single chaincode with a collection"</caption>
 	 *  [
-	 *     { name: "mychaincode", collection_names: ["mycollection"] }
+	 *     { name: "mychaincode", collectionNames: ["mycollection"] }
+	 *  ]
+	 * @example <caption>"single chaincode with a collection allowing no private data reads"</caption>
+	 *  [
+	 *     { name: "mychaincode", collectionNames: ["mycollection"], noPrivateReads: true }
 	 *  ]
 	 * @example <caption>"chaincode to chaincode with a collection"</caption>
 	 *  [
-	 *     { name: "mychaincode", collection_names: ["mycollection"] },
-	 *     { name: "myotherchaincode", collection_names: ["mycollection"] }}
+	 *     { name: "mychaincode", collectionNames: ["mycollection"] },
+	 *     { name: "myotherchaincode", collectionNames: ["mycollection"] }}
 	 *  ]
 	 * @example <caption>"chaincode to chaincode with collections"</caption>
 	 *  [
-	 *     { name: "mychaincode", collection_names: ["mycollection", "myothercollection"] },
-	 *     { name: "myotherchaincode", collection_names: ["mycollection", "myothercollection"] }}
+	 *     { name: "mychaincode", collectionNames: ["mycollection", "myothercollection"] },
```

Comment:
Is it necessary to change user interface?
from collection_names to collectionNames?

Reply:
The camel case version of the name has been used in the fabric-network to adhere to the naming convention used with that package. Both names are supported by the code, so there will be no issues for existing users. Changing the doc to reflect the name used in the high level will provide consistency between the packages.